addons/website/tests/test_performance.py

```diff
@@ -101,3 +101,9 @@ def test_40_perf_sql_queries_page_multi_level_menu(self):
         menu_aa.parent_id = menu_a
 
         self.assertEqual(self._get_url_hot_query(self.page.url), 14)
+
+    def test_50_perf_sql_web_content(self):
+        # assets route /web/content/..
+        assets_url = self.env['ir.attachment'].search([('url', '=like', '/web/content/%/web.assets_common.js')], limit=1).url
+        self._get_url_hot_query(assets_url)
+        self.assertEqual(self._get_url_hot_query(assets_url), 5)
```

Comment:
Is there a reason for calling the method twice?

Reply:
No it was leftover of dev as it was still in wip but good catch I removed it